Documentation/operations/upgrade.rst

```diff
@@ -346,10 +329,22 @@ Removed Options
   ``disable-conntrack`` option and made it non-operational. It will be removed
   in version 1.13.
 
-Deprecated Options
-~~~~~~~~~~~~~~~~~~
+Helm Options
```

Comment:
Deprecated options normal are for cli flags, so renamed this to Helm options instead.